.devcontainer/cccl-entrypoint.sh

```diff
@@ -8,6 +8,8 @@ devcontainer-utils-post-create-command;
 devcontainer-utils-init-git;
 devcontainer-utils-post-attach-command;
 
+sudo apt-get update && sudo apt-get install -y ca-certificates
```

Comment:
Could you rebase or merge with main? The other PR was merged.

Reply:
Done.